File: docs/versioned_docs/version-1.9.0/API-Reference/python-examples/workflows-api/example-request-2.py
import os

import requests

base = os.environ.get("LANGFLOW_URL") or os.environ.get("LANGFLOW_SERVER_URL", "")
flow_id = os.environ.get("FLOW_ID", "")
api_key = os.environ.get("LANGFLOW_API_KEY", "")

headers = {"Content-Type": "application/json", "x-api-key": api_key}

start = requests.post(
    f"{base}/api/v2/workflows",
    headers=headers,
    json={"flow_id": flow_id, "background": True, "stream": False, "inputs": {}},
    timeout=60,
)
start.raise_for_status()
job_id = start.json()["job_id"]

stop = requests.post(
    f"{base}/api/v2/workflows/stop",
    headers=headers,
    json={"job_id": job_id},
    timeout=60,
)
print(stop.status_code, stop.text)

File: docs/versioned_docs/version-1.11.0/API-Reference/javascript-examples/workflows-api/example-request-2.js
const url = `${process.env.LANGFLOW_SERVER_URL ?? ""}/api/v2/workflows/stop`;

const options = {
  method: 'POST',
  headers: {
    "Content-Type": `application/json`,
    "x-api-key": `${process.env.LANGFLOW_API_KEY ?? ""}`,
  },
  body: JSON.stringify({
  "job_id": "job_id_1234567890"
}),
};

fetch(url, options)
  .then(async (response) => {
    if (!response.ok) {
      throw new Error(`HTTP ${response.status}`);
    }
    const text = await response.text();
    console.log(text);
  })
  .catch((error) => console.error(error));
